File: app/src/main/java/com/justice/noteapp/Note.java
package com.justice.noteapp;

public class Note {
    private String noteData;
    private String id;
    private int category;

    public int getCategory() {
        return category;
    }

    public void setCategory(int category) {
        this.category = category;
    }

    public Note() {
    }

    public Note(String note, String id) {
        this.noteData = note;
        this.id = id;
    }

    public String getNoteData() {
        return noteData;
    }

    public void setNoteData(String noteData) {
        this.noteData = noteData;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: app/src/main/java/com/justice/noteapp/Note.kt
package com.justice.noteapp

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Note(var noteData: String,
                var id: String):Parcelable {
    var category:Int = 0
constructor() : this("dafault constructor","dafault constructor")

}